### DISCUSSION TITLE
Windows Startup and Start Minimized

### DISCUSSION THREAD

**Discussion by m567289** [Eugeny/tabby#9159] [Q&A]
Maybe I'm looking past it, but I'd love to have options for Tabby to Start on Boot and also Start Minimized.

I'm currently making it work by placing a shortcut in the Startup folder but then it pops up on my screen. I just want it to load and stay in the tray until I hit the key shortcut to show Tabby.

**Comment by tommdq** [comment]
Any update on this? I cant seem to find the usual "launch on startup option"

**Reply by Duz-Dev** [reply]
Solo tienes que precionar la combinacion de teclas `win + r` en tal ventana emergente, escribe `shell:startup` y te mostrara la carpeta donde esta alojada las aplicaciones que inician con windows. En esta carpeta coloca un acceso directo de Tabby.

**Comment by Duz-Dev** [comment]
¿Si pudiste solucionar tu duda?. Yo buscaba ayuda para el startup de tabby en windows, pero gracias a ti logre hacerlo. Mi duda ahora radica en si, ¿lograste llevar acabo lo del inicio Minimizado?. No creo que sea tarea facil si no viene nativo en tabby.
